src/parser.rs

```diff
@@ -2123,11 +2135,19 @@ impl<'input, 'abbrev, 'unit, Endian> EntriesCursor<'input, 'abbrev, 'unit, Endia
         }
     }
 
+    /// Return the current offset in bytes inside the compilation unit
+    pub fn current_offset(&self) -> usize {
```

Comment:
I don't think this should be public. As a user, I would expect this to be equivalent to `current().offset()`, but it isn't, because by the time the user can call this, `self.input` will already be pointing somewhere after the start of the current entry.